src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -458,7 +458,6 @@ private unsafe int HandleEventConnected(ref CONNECTED_DATA data)
         _connectedTcs.TrySetResult();
         return QUIC_STATUS_SUCCESS;
     }
-
```

Comment:
Are these whitespace changes intentional?

Reply:
Yes, they are. Either we do have whitespace between `Handle...` or not, but we should be consistent in all classes (Listener, Connection, Stream). I don't mind going either way, but I mind having them in Connection and nowhere else.